### ISSUE TITLE
how to solve the arm_loss=nan question?

### ISSUE THREAD

**Issue by ygren** [sfzhang15/RefineDet#84]
hello, I try to use your RefineDet to reimplement with mobilenetv2.When I was training, I had the following exception.
I0705 11:16:36.802369 11873 caffe.cpp:251] Starting Optimization
I0705 11:16:36.802381 11873 solver.cpp:294] Solving MOBILENET_V2_RefineDet
I0705 11:16:36.802386 11873 solver.cpp:295] Learning Rate Policy: multistep
I0705 11:16:38.519017 11873 solver.cpp:243] Iteration 0, loss = 288.514
I0705 11:16:38.519073 11873 solver.cpp:259]     Train net output #0: arm_loss = 157.114 (* 1 = 157.114 loss)
I0705 11:16:38.519083 11873 solver.cpp:259]     Train net output #1: odm_loss = 131.4 (* 1 = 131.4 loss)
I0705 11:16:38.519099 11873 sgd_solver.cpp:138] Iteration 0, lr = 0.001
I0705 11:16:44.513150 11873 solver.cpp:243] Iteration 4, loss = nan
I0705 11:16:44.513213 11873 solver.cpp:259]     Train net output #0: arm_loss = nan (* 1 = nan loss)
I0705 11:16:44.513224 11873 solver.cpp:259]     Train net output #1: odm_loss = 0 (* 1 = 0 loss)
How can I solve this problem? Can you give me some advice?

**Comment by sfzhang15** [comment]
@YgRen 
The loss is too large to train. Do you use the normlization layer like Conv4_3 on SSD?

**Comment by YgRen** [comment]
I reimplement your RefineDet with mobilenet and mobilenet v2, when I train with mobilenet, erveything is ok ,but mobilnet is nan, I decrease lr to 0.000001, it works, but  there are not bounding box.

**Comment by sfzhang15** [comment]
@YgRen 
You should use the normlization layer to make the loss value about 10.

**Comment by YgRen** [comment]
ok,I will try it again with mobienetv2.If everything is ok, I will release the RefineDet implement with mobilenet and mobilentv2 .

**Comment by sfzhang15** [comment]
@YgRen 
You should try to use the normlization layer for the detection layers to make the loss value about 10~20.

**Comment by litingsjj** [comment]
@YgRen Have you finish the work with mobilenetV2? I meet the same problem with mobilenetV2, and about mobilenetV1, the loss=10~11

**Comment by campos537** [comment]
> ok,I will try it again with mobienetv2.If everything is ok, I will release the RefineDet implement with mobilenet and mobilentv2 .

hey man , was you able to get good result with mobilenet?
